src/components/Common/TitleAndList.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import remcalc from 'remcalc'
+import ReactMarkdown from 'react-markdown'
+import styled from 'styled-components'
+import { Col, Row } from '../grid'
+import { Padding } from 'styled-components-spacing'
+import { Subtitle, BodyPrimary, SectionTitle } from '../Typography'
+
+const OuterPaddings = ({ children }) => (
+  <Padding vertical={{ smallPhone: 3, tablet: 4 }}>{children}</Padding>
+)
+
+const ParagraphWithOptionalPadding = styled(BodyPrimary)`
+  padding-bottom: ${remcalc(24)};
+  &:last-child {
+    padding: 0;
+  }
+`
+const ItemTitle = ({ children, ...props }) => (
+  <Subtitle as="h3" noPadding bold {...props}>
+    {children}
```

Comment:
not sure what you mean. I created this component to avoid repeating the `as`, `noPadding` and `bold` props. As there's already a `Subtitle` in the scope (imported one), I decided `ItemTitle` is a good name for what it does.

Reply:
See above comment - we can remove all these props :+1:

Reply:
Oh woops. Resolved. My bad

Reply:
we have to keep the `as`,  since `Subtitle` is an h4, right?